stripe/_stripe_client.py

```diff
@@ -280,3 +286,21 @@ def construct_event(
         )
 
         return event
+
+    def raw_request(self, method_: str, url_: str, **params):
+        return _raw_request(method_, url_, **params)
+
+    async def raw_request_async(self, method_: str, url_: str, **params):
+        return await _raw_request_async(method_, url_, **params)
+
+    def deserialize(
+        self,
+        resp: Union[StripeResponse, Dict[str, Any]],
+        api_key: Optional[str] = None,
```

Comment:
one question I had here: since we have access to the API key as part of StripeClient, do we still want to take api_key as an argument?  `deserialize` in other languages (i.e. java) don't; here, it's used to construct a requestor that is used in object construction during deserialization (I can assume so that some objects can make subsequent API calls?)